sphinx_gallery/gen_rst.py

```diff
@@ -138,8 +138,7 @@ def __exit__(self, type_, value, tb):
     :class: sphx-glr-single-img
 """
 
-# This one could contain unicode
-CODE_OUTPUT = u""".. rst-class:: sphx-glr-script-out
+CODE_OUTPUT = """.. rst-class:: sphx-glr-script-out
```

Comment:
Why did we allow unicode in the first place...?

Reply:
We still do implicitly because you just don't need `u` in Python3